Formula/thrift.rb

```diff
@@ -59,6 +65,7 @@ def install
     ENV.deparallelize
     system "make"
     system "make", "install"
+    libexec.install Dir.glob(buildpath/"usr/local/lib/*.jar") if build.with? "java"
```

Comment:
83rd column. Are you sure you're feeling all right?

Reply:
It's one of those edge cases. Do I hate 80+ characters more than I hate creating a whole block simply to avoid 83 characters 😓.